try.html

```diff
@@ -553,6 +553,10 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
   <td><img src='/dub/l/vibe-d.svg' alt=''/></td>
   <td><code>https://img.shields.io/dub/l/vibe-d.svg</code></td>
   </tr>
+  <tr><th data-keywords='docker stars'> Docker Stars: </th>
+  <td><img src='/docker/stars/mashape/mockbin.svg' alt=''/></td>
```

Comment:
Actually I think Docker registry shut off public access to the API I was using, so this integration no longer works. The original ticket #415 creator @nazar-pc posted in the docker repo asking for access to downloads in addition to stars, so maybe they will expose this in a new public API in the future.